tests/testCameraMapper.py

```diff
@@ -52,6 +52,9 @@ def __init__(self):
         lsst.obs.base.CameraMapper.__init__(self, policy=policy, repositoryDir=testDir, root=testDir)
         return
 
+    @classmethod
+    def getPackageDir(cls):
+        return "/path/to/nowhere"
```

Comment:
It's defined in the base class, a shorthand for `lsst.utils.getPackageDir(MapperClass.getPackageName())`.

I redefine it for the test mappers because they don't define `MapperClass.getPackageName` or `MapperClass.packageName`, and even if they did there's no recipe override for them to grab.